### ISSUE TITLE
How did you split the train dataset and test dataset for ffhq inversion？

### ISSUE THREAD

**Issue by wtybest** [genforce/idinvert_pytorch#17]
How did you split the train dataset and test dataset for ffhq inversion？

**Comment by zhujiapeng** [comment]
Take the first 65000 images as the training set and take the rest as the test set.
